apps/passport-server/src/services/issuanceService.ts

```diff
@@ -522,6 +624,84 @@ export class IssuanceService {
     }
   }
 
+  /**
+   * Checks that a ticket is valid for check-in based on the ticket data in
+   * the DB.
+   */
+  public async handleCheckTicketByIdRequest(
+    request: CheckTicketByIdRequest
+  ): Promise<CheckTicketByIdResult> {
+    try {
+      const ticketId = request.ticketId;
+      return this.checkTicketById(ticketId);
+    } catch (e) {
+      return {
+        error: { name: "ServerError" },
+        success: false
+      };
+    }
+  }
+
+  /**
+   * Checks a ticket for validity based on the ticket's status in the DB.
+   */
+  public async checkTicketById(
```

Comment:
same here - this is only for devconnect tickets - worth communicating that in comment and variable name